app/models/manager_refresh/inventory_collection.rb

```diff
@@ -35,6 +36,7 @@ def initialize(model_class: nil, manager_ref: nil, association: nil, parent: nil
       @complete                         = complete.nil? ? true : complete
       @update_only                      = update_only.nil? ? false : update_only
       @builder_params                   = builder_params
+      @inventory_object_attributes      = inventory_object_attributes
```

Comment:
hm, weird, should be enough to put there a one line with raise

I am still not convinced I want to necessarily list all attributes manually, when I can introspect them. I will need to see how it looks in action and the benefits I suppose. :-)